Add Scout self-serve billing and subscription pages (#17655)
docker/docs: subscription/scout-details.md

Reviewer: **though:** maybe we should rename `Docker subscriptions and features` to `Docker XYZ subscriptions and features`. 

Otherwise, it might look generic and confuse people when they see:

<img width="300" alt="Screenshot 2023-06-30 at 10 34 10 AM" src="https://github.com/docker/docs/assets/2487951/e20bea9c-5135-4f33-82a6-0242d1922390">

@yavorg how do we call Hub+Desktop subscriptions?

Author: Good idea to keep in mind for this section of the ToC for the future. Open to hearing naming ideas and can raise the question with other docs writers as well!

---
title: Docker Scout subscriptions and features
description: Learn about the Docker Scout subscriptions tiers and features
keywords: subscription, free, team, business, features
---

{% include scout-early-access.md %}

This page provides an overview of each subscription tier for Docker Scout. To compare features available in each tier, see [Docker Scout Pricing](https://docker.com/products/docker-scout){: target="_blank" rel="noopener" class="_" id="dkr_docs_scout_subscription_btl"}.

## Docker Scout Free 

Docker Scout Free is available for both personal accounts and organizations. This tier is ideal for individual developers getting started with software supply chain security.

Docker Scout Free includes:

- Unlimited local image analysis

> - Remote image analysis for up to 3 repositories

Reviewer 2: This bullet should probably move last

```suggestion
- Up to 3 Scout-enabled repositories
```

- Vulnerability analysis
- Image comparison for vulnerabilities, size, and packages
- Recommendations for refreshing or updating the base image

> - SDLC integration

Reviewer 2: ```suggestion
- SDLC integration (Policy evaluation and Workload integration)
- On-prem and cloud container registry integrations
```

- Security posture reporting

## Docker Scout Team

Docker Scout Team is available for both personal accounts and organizations. This plan is ideal for development teams developing and maintaining multiple container images. 

Docker Scout Team includes:

- All the features available in Docker Scout Free

> - Remote image analysis for up to 100 repositories

Reviewer 2: ```suggestion
- up to 100 Scout-enabled repositories
```

You can buy Docker Scout Team in Docker Hub. See [Docker Scout billing](../billing/scout-billing.md).

## Docker Scout Business

Docker Scout Business is ideal for commercial organizations managing multiple teams. 

Docker Scout Business includes:

- All the features available in other Docker Scout tiers

> - Remote image analysis in unlimited repositories

Reviewer 2: ```suggestion
- Unlimited Scout-enabled repositories
```

---
title: Docker Scout subscriptions and features
description: Learn about the Docker Scout subscriptions plans and features
keywords: subscription, free, team, business, features
---

{% include scout-early-access.md %}

> This page provides an overview of the subscription plans for Docker Scout. To compare features available for each plan, see [Docker Scout Pricing](https://docker.com/products/docker-scout){: target="_blank" rel="noopener" class="_" id="dkr_docs_scout_subscription_btl"}.

Author: **Note for docs reviewer:** what is the id format for these links? I modified a similar one from the subscription page for this but didn't see a process documented for how we come up with these. Would love your input!

Reviewer 3: we're actually a bit naughty and don't use that...When we move to hugo all those jekyll tags will be removed anyway, so no need to worry about them

Reviewer 4: I think we just add them to the Url for now? Like…

```suggestion
This page provides an overview of the subscription plans for Docker Scout. To compare features available for each plan, see [Docker Scout Pricing](https://docker.com/products/docker-scout?id=dkr_docs_scout_subscription_btl){: target="_blank" rel="noopener" class="_"}.
```

Reviewer 5: What's the ID for? the first example would add an HTML ID attribute to the <a> element. The example @ChrisChinchilla gave would pass the ID as a URL parameter on the HTTP request. There's also a third option of including the ID as an anchor link, and you can do that by simply appending it to the URL:

`https://docker.com/products/docker-scout#dkr_docs_scout_subscription_btl`

But I didn't see any such ID on the target page so I don't think that would be necessary.

Reviewer 4: I assumed it was some form of tracking ID, but I have not seen this one before… Links from Hub and desktop and up with this full string:

https://www.docker.com/products/docker-scout/?utm_source=hub&utm_content=own-tag-details-banner

So maybe

```suggestion
This page provides an overview of the subscription plans for Docker Scout. To compare features available for each plan, see [Docker Scout Pricing](https://www.docker.com/products/docker-scout/?utm_source=docs){: target="_blank" rel="noopener" class="_"}.
```

?

But perhaps the tag @stephaurelio added has som meaning we're not aware of…

Author: I noticed this was how the link to a pricing page was formatted on the core subscription page so was wondering if we need to do these for all links like this but it sounds like maybe that was a one-off and I can remove this?

Reviewer 4: @stephaurelio I am not sure, so if we're not certain and have no explicit instruction, let's remove it then

## Docker Scout Free 

Docker Scout Free is available for both personal accounts and organizations. This plan is ideal for individual developers getting started with software supply chain security.

Docker Scout Free includes:

- Unlimited local image analysis
- Up to 3 Docker Scout-enabled repositories
- Vulnerability analysis
- Image comparison for vulnerabilities, size, and packages
- Recommendations for refreshing or updating the base image
- SDLC integration, including policy evaluation and workload integration
- On-prem and cloud container registry integrations

> - Security posture reporting

Reviewer: **suggestion:** Looks like the list of features has been trimmed.

<img width="1306" alt="Screenshot 2023-06-30 at 10 26 48 AM" src="https://github.com/docker/docs/assets/2487951/2d843d92-5922-4409-9ef3-968c390a2594">

@cdupuis could you please confirm?

Author: Confirmed on Slack to update docs to match the trimmed list: https://docker.slack.com/archives/C04C69EM70C/p1688152892696629?thread_ts=1688022705.075599&cid=C04C69EM70C

## Docker Scout Team

Docker Scout Team is available for both personal accounts and organizations. This plan is ideal for development teams developing and maintaining multiple container images. 

Docker Scout Team includes:

- All the features available in Docker Scout Free
- Up to 100 Docker Scout-enabled repositories

You can buy Docker Scout Team in Docker Hub. See [Docker Scout billing](../billing/scout-billing.md).

## Docker Scout Business

Docker Scout Business is ideal for commercial organizations managing multiple teams. 

Docker Scout Business includes:

> - All the features available in other Docker Scout plans

Reviewer: **suggestion:** I would highlight here that it's better than Scout Team tier.

```suggestion
- All the features available in Docker Scout Team
```
